docs/ECLStandardLibraryReference/SLR-Mods/GetLogicalFileAttribute.xml

```diff
@@ -31,7 +31,11 @@
           <entry><emphasis>attrname</emphasis></entry>
 
           <entry>A null-terminated string containing the name of the file
-          attribute to return.</entry>
+          attribute to return. Possible values are recordSize, recordCount,
+          size, clusterName, directory, numparts, owner, description, ECL,
+          partmask, numparts, name, modified, format, job, checkSum, kind,
+          csvSeparate, csvTerminate, headerLength, footerLength, rowTag,
+          workunit, or protected. The value is case-sensitive.</entry>
```

Comment:
I found some more: 
- accessed
- maxRecordSize
- csvQuote (for CSV file if it is defined for the command)
- blockCompressed, compressedSize (if the file compressed) 

The 'numparts' is duplicated: